docs/field-details.asciidoc

```diff
@@ -353,13 +353,15 @@ example: `184`
 [[field-client-domain]]
 <<field-client-domain, client.domain>>
 
-| Client domain.
+| The domain name of the client system.
 
-type: keyword
+This value may be a host name, a fully qualified domain name, or other host naming format. The value may from the original event or added from enrichment.
```

Comment:
`The value may from the original event ` might be better phrased as 

The value may derive from the original event `